client/components/boards/boardsList.jade

```diff
@@ -26,8 +26,6 @@ template(name="boardList")
 
 
 template(name="boardListHeaderBar")
-  h1 {{_ 'my-boards'}}
```

Comment:
I would say, keep it on desktop and hide it on mobile. This is the visual position where we put a title on all pages.